docs/guides/stay-up-to-date/terraform/terraform-15/deployment-walkthrough/step-1-update-your-code-to-be-compatible-with-terraform-0-14.md

```diff
@@ -10,19 +10,19 @@ If you haven’t already, you need to:
     0.14.0 or above first!
 
     1.  If you’re still on Terraform 0.11 or older, see our
-        [Terraform 0.12 upgrade guide](../../terraform-12).
+        [Terraform 0.12 upgrade guide](../../terraform-12/index.md).
 
     2.  If you’re still on Terraform 0.12, see our
-        [Terraform 0.13 upgrade guide](../../terraform-13).
+        [Terraform 0.13 upgrade guide](../../terraform-13/index.md).
 
     3.  If you’re still on Terraform 0.13, see our
-        [Terraform 0.14 upgrade guide](../../terraform-14).
+        [Terraform 0.14 upgrade guide](../../terraform-14/index.md).
```

Comment:
Do we need the `/index.md` on these? That sort of eliminates the advantage of using the index convention.

Reply:
We discussed this live - but for posterity: Yes - we do need this otherwise we get the 404 when the site is hosted in S3. Notably, the final URL (rendered into HTML) will not have `/index.md` in it.